presto-hive/src/main/java/io/prestosql/plugin/hive/azure/HiveAzureConfig.java

```diff
@@ -118,4 +121,43 @@ public HiveAzureConfig setAdlRefreshUrl(String adlRefreshUrl)
         this.adlRefreshUrl = adlRefreshUrl;
         return this;
     }
+
+    @ConfigSecuritySensitive
+    @Config("hive.azure.abfs.oauth-client-endpoint")
```

Comment:
Add a dot after `oauth` since this is a separate group
```
hive.azure.abfs.oauth.client-endpoint
hive.azure.abfs.oauth.client-id
hive.azure.abfs.oauth.client-secret

Reply:
I hesitated to do that because it's inconsistent with the existing configuration properties, which I assume we don't want to change.